upload_parrots.py

```diff
@@ -19,17 +20,35 @@ def __init__(self, slack_team):
         self.browser = webdriver.Chrome()
         self.slack_team = slack_team
 
-    def login(self, username, password):
+    def login(self, username, password, google):
         if self.loggged_in:
             return
 
         self.browser.get('https://{}.slack.com/signin'.format(self.slack_team))
-        username_field = self.browser.find_element(value="email")
-        password_field = self.browser.find_element(value='password')
 
-        username_field.send_keys(username)
-        password_field.send_keys(password)
-        username_field.submit()
+        if google == True:
```

Comment:
No biggie, but a more pythonic way to write this is:
```python
if google:
...
else:
...
```

in the case of this arg, as `store_true` ensures this will come as a boolean, the `google` arg can really only take the shape of `True` or `False`, so other options would never happen